### PR TITLE
CommTrack -> CommCare Supply

### PR DESCRIPTION
**Pull request by esoergel**
Rename a bunch of display strings from "CommTrack" to "CommCare Supply".  This has been on staging for a while.
@proteusvacuum 
@nickpell I imagine you'll be happy to see this merged.

### CODE REVIEW

File: corehq/apps/accounting/forms.py [hunk 1]
```diff
@@ -558,11 +558,11 @@ def clean(self):
                 )
             ):
                 from corehq.apps.accounting.views import ManageBillingAccountView
-                raise forms.ValidationError(mark_safe(
+                raise forms.ValidationError(mark_safe(_(
                     "Please update 'Client Contact Emails' "
                     '<strong><a href=%s target="_blank">here</a></strong> '
                     "before using Billing Account <strong>%s</strong>."
-                    % (
+                    ) % (
```
Comment (dimagimon) [inline corehq/apps/accounting/forms.py]: E123 closing bracket does not match indentation of opening bracket's line

File: corehq/apps/accounting/forms.py [hunk 2]
```diff
@@ -571,10 +571,10 @@ def clean(self):
         start_date = self.cleaned_data.get('start_date') or self.subscription.date_start
         if (self.cleaned_data['end_date'] is not None
             and start_date > self.cleaned_data['end_date']):
-            raise ValidationError("End date must be after start date.")
-
+            raise ValidationError(_("End date must be after start date."))
+
```
Comment (dimagimon) [inline corehq/apps/accounting/forms.py]: W293 blank line contains whitespace

File: corehq/apps/sms/forms.py [hunk 3]
```diff
@@ -854,7 +854,7 @@ class SubscribeSMSForm(Form):
     non_report = BooleanField(
         label=ugettext_lazy("Receive non-reporting SMS alert"),
         required=False,
-        help_text=ugettext_lazy("This alert highlight users/facilities which have not submitted their CommTrack stock report.")
+        help_text=ugettext_lazy("This alert highlight users/facilities which have not submitted their CommCare Supply stock report.")
```
Comment (dimagimon) [inline corehq/apps/sms/forms.py]: E501 line too long (133 > 115 characters)

File: corehq/apps/reports/__init__.py [hunk 4]
```diff
@@ -9,4 +9,4 @@
 _("Inspect Data")
 _("Raw Data")
 _("Manage Deployments")
-_("Commtrack")
+_("CommTrack")
```
Comment (proteusvacuum) [inline corehq/apps/reports/__init__.py]: Should this be `CommCare Supply`?

Comment (esoergel) [inline corehq/apps/reports/__init__.py]: Yes and no - this is just here to be marked for translation, and my thoughts were that if this string IS found, it should be translated to "CommCare Supply".  I've added a comment to clarify.

Comment (proteusvacuum) [inline corehq/apps/reports/__init__.py]: :+1:

**Comment by esoergel** [comment]
@benrudolph ping

File: corehq/apps/style/static/style/lib/select2 [hunk 5]
```diff
@@ -1 +1 @@
-Subproject commit 6633cda9061d9fc65075dd140c63f9a6adbcbd72
+Subproject commit 3a10bd997a4d6a4a54e4bd4ebc210f51af0944b1
```
Comment (czue) [inline corehq/apps/style/static/style/lib/select2]: is this intentional?

Comment (esoergel) [inline corehq/apps/style/static/style/lib/select2]: No, I'm guessing it came in during a merge.  https://github.com/dimagi/commcare-hq/pull/6598

**Pull request merged**